lambda_functions/two_way_sms/two_way_sms_v2.py

```diff
@@ -0,0 +1,423 @@
+import boto3
+from datetime import datetime
+import json
+import logging
+import os
+import psycopg2
+import requests
+import sys
+
+# Global Variables
+# Constants
+START_TYPES = ('START', 'BEGIN', 'RESTART', 'OPTIN', 'OPT-IN',)
+STOP_TYPES = ('STOP', 'OPTOUT', 'OPT-OUT',)
+HELP_TYPES = ('HELP',)
+START_TEXT = 'Message service resumed, reply "STOP" to stop receiving messages.'
+STOP_TEXT = 'Message service stopped, reply "START" to start receiving messages.'
+HELP_TEXT = 'Some help text'
+INBOUND_NUMBERS_QUERY = """SELECT number, service_id, url_endpoint, self_managed FROM inbound_numbers;"""
+# Validation set
+EXPECTED_PINPOINT_FIELDS = set({'originationNumber',
+                           'destinationNumber',
+                           'messageBody'
+                           })
+
+# Pre-defined env variables
+AWS_PINPOINT_APP_ID = ''
+AWS_REGION = ''
+DEAD_LETTER_SQS_URL = ''
+LOG_LEVEL = 'INFO'
+RETRY_SQS_URL = ''
+SQLALCHEMY_DATABASE_URI = ''
+DATABASE_URI_PATH = ''
+TIMEOUT = 3
+
+logger = logging.getLogger('TwoWaySMSv2')
+logger.setLevel(logging.getLevelName(LOG_LEVEL))
+
+# Set within lambda
+aws_pinpoint_client = None
+aws_sqs_client = None
+two_way_sms_table_dict = {}
+initialized = False
+
+
+def set_env_variables() -> None:
+    """
+    Attempt to get environmental variables. Abort execution environment on failure.
+    """
+    global AWS_PINPOINT_APP_ID, AWS_REGION, DEAD_LETTER_SQS_URL
+    global LOG_LEVEL, RETRY_SQS_URL, DATABASE_URI_PATH, TIMEOUT
+
+    try:
+        AWS_PINPOINT_APP_ID = os.getenv('AWS_PINPOINT_APP_ID')
+        AWS_REGION = 'us-gov-west-1'
```

Comment:
Since this was changed to a constant you can remove it from here and line 49, updating it on line 27.

Reply:
done